packages/google_maps_flutter/google_maps_flutter/ios/Classes/JsonConversions.m

```diff
@@ -68,4 +68,43 @@ + (UIColor *)toColor:(NSNumber *)numberColor {
   return holes;
 }
 
++ (GMUWeightedLatLng *)toWeightedLatLng:(NSArray *)data {
+  return
```

Comment:
How are we guaranteeing the length of data is 2?

Reply:
I suppose we aren't, but I'm not sure the other conversions are checking for this either

Reply:
What do you want to do about this one?

Reply:
Maybe something like:

```objc
NSAssert(data.count == 2);
if (data.count != 2) {
   return nil;
}
```

Reply:
Sorry I'm learning Objective-C as I go here. When I try to use NSAssert I get `Use of undeclared identifier 'NSAssert'`. I tried importing Foundation, but that didn't fix it.

Reply:
Finally figured it out